_includes/banner-info.html

```diff
@@ -3,10 +3,10 @@
     <div class="usa-alert">
       <div class="usa-alert__body">
         <h3 class="usa-alert__heading text-black">
-          Header here
+          Delay in processing domain requests
```

Comment:
I think I'd rather say something plainer, like "Our review is taking longer than normal". I defer to the team.

Reply:
@Katherine-Osos and I like it! Updated and merged!